code/modules/spells/spell_types/vampire_powers.dm

```diff
@@ -88,7 +88,12 @@
 /obj/effect/proc_holder/spell/self/rejuvenate/cast(list/targets, mob/user = usr)
 	var/mob/living/carbon/U = user
 	U.stuttering = 0
-	U.do_adrenaline(0, TRUE, 0, 0)
+	U.SetSleeping(0)
+	U.SetUnconscious(0)
+	U.resting = 0
+	U.lying = 0
+	U.update_canmove()
```

Comment:
I foresee quirky interactions between this and stamcrit.

Reply:
Havent finnished it yet, Should add a bit of instant stamina regen to null out stamcrit. Hopefully.